### PR TITLE
Accessibility improvements

### LINKED ISSUE #2919: A11y: Tab order lands on element "Pane" with no focus indicator
**Describe the bug**
When opening a folder and tabbing through the UI, I noticed that focus can land on an element that is being announced "Pane" without a more descriptive name that also has no focus visual:

**To Reproduce**
1. Open folder in Files (in my case desktop)
2. Tab through entire app
3. Start narrator
4. Notice elements visited and announced
4. Notice pane without name or focus indicator

**Expected behavior**
Either give it a proper name and add focus indicator or skip element in tab order
**Screenshots**
Gif showing faulty tab order where focus rect moves through app and then suddenly disappears

**Desktop (please complete the following information):**
 - OS Version: Win10 10.0.19042
 - App version: [1.0.0.0]

**Additional context**
Add any other context about the problem here. Does this problem occur again after restarting the app?

**Log file**
Please upload the log file here so that we can understand your issue better. You can access it from Settings->About->Open log location. You can also find the log file manually by following [this](https://github.com/files-community/Files/wiki/Opening-the-log-file) guide.
[debug_fulltrust.txt](https://github.com/files-community/Files/files/5797991/debug_fulltrust.txt)
[debug.txt](https://github.com/files-community/Files/files/5797992/debug.txt)

### PR DESCRIPTION
**Pull request by winston-de**
This PR closes a few accessibility issues from the to-do list.
Closes #2919, #2577, and #2563.

Also added "not calculated" text when there is no file size, as requested in #2570.
I also did the groundwork for labeling the sync status, but I have been unable to get automation properties to work with the DataGrid.

### CODE REVIEW

File: Files/UserControls/StatusBarControl.xaml [hunk 1]
```diff
@@ -46,7 +46,7 @@
             x:Uid="StatusCenterTeachingTip"
             Title="Status Center"
             Subtitle="Check the status of file operations here"
-            Target="{x:Bind StatusCenter}" />
```
Comment (yaichenbaum) [inline Files/UserControls/StatusBarControl.xaml:49]: Was this property removed on purpose?

File: Files/Views/LayoutModes/GenericFileBrowser.xaml.cs [hunk 2]
```diff
@@ -337,11 +337,13 @@ private void TextBox_TextChanged(object sender, TextChangedEventArgs e)
 
             if (FilesystemHelpers.ContainsRestrictedCharacters(textBox.Text))
             {
+                FileNameTeachingTip.Visibility = Visibility.Visible;
                 FileNameTeachingTip.IsOpen = true;
             }
             else
             {
                 FileNameTeachingTip.IsOpen = false;
+                FileNameTeachingTip.Visibility = Visibility.Collapsed;
```
Comment (tsvietOK) [inline Files/Views/LayoutModes/GenericFileBrowser.xaml.cs:346]: What is the purpose of this change? I think setting `IsOpen` to false is enough to hide tip.

File: Files/Views/LayoutModes/GenericFileBrowser.xaml [hunk 3]
```diff
@@ -980,7 +983,7 @@
                     x:Uid="sizeColumn"
                     Width="Auto"
                     MinWidth="100"
-                    Binding="{Binding FileSize}"
+                    Binding="{Binding FileSizeDisplay}"
```
Comment (yaichenbaum) [inline Files/Views/LayoutModes/GenericFileBrowser.xaml:986]: Perhaps the AutomationProperties name should say "Not Calculated" and the size should remain blank.

File: Files/Views/LayoutModes/GenericFileBrowser.xaml.cs (hunk 2 above)
Comment (winston-de) [reply Files/Views/LayoutModes/GenericFileBrowser.xaml.cs:346]: @tsvietOK If you don't collapse it, it shows up on the tab stack. This change fixes #2919

File: Files/Views/LayoutModes/GenericFileBrowser.xaml (hunk 3 above)
Comment (winston-de) [reply Files/Views/LayoutModes/GenericFileBrowser.xaml]: @yair100 I was unsuccessful in getting automation properties to work with the DataGrid. The screen reader wouldn't read out the name when the column was selected. I had the same issue with trying to add a name to the sync status glyph. If you want, I could switch back and we can try to implement this at a later date (Perhaps when the DataGrid is modernized?).

Comment (yaichenbaum) [reply Files/Views/LayoutModes/GenericFileBrowser.xaml:986]: Let's to that and wait on the new DataGrid control to fix this issue.

File: Files/Filesystem/ListedItem.cs [hunk 4]
```diff
@@ -138,6 +138,7 @@ public string ItemType
 
         public string FileExtension { get; set; }
         public string FileSize { get; set; }
+        public string FileSizeDisplay => FileSize ?? "ItemSizeNotCalcluated".GetLocalized();
```
Comment (yaichenbaum) [inline Files/Filesystem/ListedItem.cs:141]: Can you remove this string?

File: Files/Strings/en-US/Resources.resw [hunk 5]
```diff
@@ -1788,4 +1788,28 @@
...
+  </data>
+  <data name="CloudDriveSyncStatus_Sync" xml:space="preserve">
+    <value>Syncing</value>
+  </data>
+  <data name="CloudDriveSyncStatus_Excluded" xml:space="preserve">
+    <value>Excluded from sync</value>
+  </data>
+  <data name="ItemSizeNotCalcluated" xml:space="preserve">
+    <value>Not calculated</value>
+  </data>
```
Comment (yaichenbaum) [inline Files/Strings/en-US/Resources.resw:1811]: Can you remove this string?

Comment (winston-de) [reply Files/Strings/en-US/Resources.resw:1811]: @yair100 Shouldn't this be kept for when #2570 is eventually able to be implemented?

**Review by yaichenbaum — APPROVED** [review]

**Comment by yaichenbaum** [comment]
@winston-de Great work! Accessibility is a high priority for Files, so these kinds of pull requests help Files stay true to its vision.
